### PR TITLE
[chart] change metaflow to deepflow

### PR DESCRIPTION
**Pull request by mytting**
**Phenomenon and reproduction steps**

change metaflow to deepflow

**Root cause and solution**

**Impactions**

**Test method**

**Affected branch(es)**

* main

**Checklist**

- [ ] Dependencies update required
- [ ] Common bug (similar problem in other repo)

### CODE REVIEW

File: manifests/charts/deepflow-agent/values.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ global:
 
 
 image:
-  repository: registry.cn-beijing.aliyuncs.com/deepflowys/deepflow-agent
+  repository: deepflowce/deepflow-agent
```
Comment (sharang) [inline manifests/charts/deepflow-agent/values.yaml:14]: 能加减号吗

Comment (sharang) [reply manifests/charts/deepflow-agent/values.yaml:14]: deepflow-ce

File: manifests/charts/deepflow/values.yaml [hunk 2]
```diff
@@ -3,10 +3,10 @@
...
 global:
   image:
-    ## ghcr Image repository address: ghcr.io/deepflowys
-    ## Dockerhub Image repository address: deepflowys
-    ## AliyunYun Image repository address: registry.cn-beijing.aliyuncs.com/deepflowys
-    repository: deepflowys
+    ## ghcr Image repository address: ghcr.io/deepflowys/deepflow-ce
+    ## Dockerhub Image repository address: deepflowce
+    ## AliyunYun Image repository address: registry.cn-beijing.aliyuncs.com/deepflow-ce
+    repository: deepflowce
```
[the remark this answers is not in the document]
Comment (mytting) [reply manifests/charts/deepflow/values.yaml:9]: dockerhub的用户名，无法带减号

File: manifests/charts/deepflow-agent/values.yaml (hunk 1 above)
Comment (mytting) [reply manifests/charts/deepflow-agent/values.yaml:14]: dockerhub的用户名，无法带减号

**Pull request merged**